compile.sh

```diff
@@ -147,7 +147,7 @@ pso_dir=${project_root}/scripts/pso
 rm -rf ${project_root}/dist
 mkdir dist
 
-play_command $OPTS clean test compile dist
+play_command $OPTS clean compile dist
```

Comment:
can you re-enable the test?

Reply:
Woops yeah this wasn't supposed to make it into the changelist. I reverted in https://github.com/linkedin/dr-elephant/pull/378/commits/068ff1a804db95abb4ee5b0b5f5d772815fbee6b.